lecture_2/fh2386/fh2386.tex

```diff
@@ -0,0 +1,129 @@
+%----------------------------------------
+% Write your notes here
+%----------------------------------------
+\section{Part 1}
+\subsection{Conduct 5\% margin of error}
+\begin{documents}
+\ITEM{\textbf{5\% margin}} margin of error means 5\% standard error or 5\% standard deviation in sampling distribution for the mean.
+\\\\
+\ITEM{\textbf{Given 5\%}} margin of error, 100 samples are needed. However,if the standard error is large such as 10\%, less samples will need. Following R codes has provided to demonstrate.\\\\
+\ \# flip n times p=0.5
+\\rbinom(n,1,p)\\
+\#estimate p by measuring the fraction of heads \\
+mean(rbinom(n,1,p)\\
+\# repeat this 100,000 times\\
+ p\_hat\<-\ replicate(1e5,mean(rbinom(n,1,p))\\
```

Comment:
Here's how you can include verbatim R code in LaTeX without having to escape things:

https://www.overleaf.com/learn/latex/Questions/How_do_I_include_sections_of_R_code_within_my_LaTeX_document%3F